src/test/java/stage1/RunnerSprintNameGeneratorTest.java

```diff
@@ -0,0 +1,291 @@
+		}
+		SprintName.setVoteCount(4);
+		RunnerSprintNameGenerator.getSprintNames().get(1).voteUp();
+		RunnerSprintNameGenerator.getSprintNames().get(1).voteUp();
+		RunnerSprintNameGenerator.getSprintNames().get(2).voteUp();
+		RunnerSprintNameGenerator.getSprintNames().get(2).voteUp();
+		assertEquals(RunnerSprintNameGenerator.getSprintNames().get(1), RunnerSprintNameGenerator.getNamesWithMostVotes().get(0));
+		assertEquals(RunnerSprintNameGenerator.getSprintNames().get(2), RunnerSprintNameGenerator.getNamesWithMostVotes().get(1));
+	}
+
+	@Test
+	public void test_getHighestVotes_checkIfHighestVotesIsCorrect() {
+		setTestSprintNames();
+		for (SprintName sprintName : RunnerSprintNameGenerator.getSprintNames()) {
+			sprintName.displaySprintName();
+			sprintName.displayVoteButton();
+		}
+		SprintName.setVoteCount(5);
+		RunnerSprintNameGenerator.getSprintNames().get(1).voteUp();
+		RunnerSprintNameGenerator.getSprintNames().get(1).voteUp();
+		RunnerSprintNameGenerator.getSprintNames().get(2).voteUp();
+		RunnerSprintNameGenerator.getSprintNames().get(2).voteUp();
+		RunnerSprintNameGenerator.getSprintNames().get(2).voteUp();
+		assertEquals(3, RunnerSprintNameGenerator.getHighestVotes());
+	}
+
+
+	@Test
+	public void test_getHighestVotes_checkIfHighestVotesIsCorrectEvenIfMoreNamesHasTheSameVotes() {
+		setTestSprintNames();
+		for (SprintName sprintName : RunnerSprintNameGenerator.getSprintNames()) {
+			sprintName.displaySprintName();
+			sprintName.displayVoteButton();
+		}
+		SprintName.setVoteCount(6);
+		RunnerSprintNameGenerator.getSprintNames().get(1).voteUp();
+		RunnerSprintNameGenerator.getSprintNames().get(1).voteUp();
+		RunnerSprintNameGenerator.getSprintNames().get(1).voteUp();
+		RunnerSprintNameGenerator.getSprintNames().get(2).voteUp();
+		RunnerSprintNameGenerator.getSprintNames().get(2).voteUp();
+		RunnerSprintNameGenerator.getSprintNames().get(2).voteUp();
+		assertEquals(3, RunnerSprintNameGenerator.getHighestVotes());
+	}
+
+	@Test
+	public void test_startRevote_checkIfRevoteStartsWithRemainingNames() {
+		setTestSprintNames();
+		for (SprintName sprintName : RunnerSprintNameGenerator.getSprintNames()) {
+			sprintName.displaySprintName();
+			sprintName.displayVoteButton();
+		}
+		SprintName.setVoteCount(6);
+		RunnerSprintNameGenerator.getSprintNames().get(1).voteUp();
+		RunnerSprintNameGenerator.getSprintNames().get(1).voteUp();
+		RunnerSprintNameGenerator.getSprintNames().get(1).voteUp();
+		RunnerSprintNameGenerator.getSprintNames().get(2).voteUp();
+		RunnerSprintNameGenerator.getSprintNames().get(2).voteUp();
+		RunnerSprintNameGenerator.getSprintNames().get(2).voteUp();
+		RunnerSprintNameGenerator.startRevote(RunnerSprintNameGenerator.getNamesWithMostVotes());
+		assertEquals(1, RunnerSprintNameGenerator.getSprintNames().get(0).getIndex());
+		assertEquals(2, RunnerSprintNameGenerator.getSprintNames().get(1).getIndex());
+	}
+	
+	@Test
+	public void test_markNamesDraw_checkIfNamesWereHighlightedYellow() {
```

Comment:
UnitTest not successful

Reply:
I haven't checked due to time yet, but I am pretty sure that this is due to the same issue as in test `test_markWinnerName_checkIfCorrectNameIsHighlightedGreen`, so it should be resolvable via the same means.